compiler-rt/lib/scudo/standalone/tests/combined_test.cpp

```diff
@@ -54,7 +54,7 @@ void checkMemoryTaggingMaybe(AllocatorT *Allocator, void *P, scudo::uptr Size,
     EXPECT_DEATH(
         {
           disableDebuggerdMaybe();
-          reinterpret_cast<char *>(P)[-1] = 0xaa;
+          reinterpret_cast<char *>(P)[-1] = '\xaa';
```

Comment:
Let's just use 'A' (this is just a crash test, so the value doens't matter).

Reply:
Done.